pontoon/base/tests/test_placeables.py

```diff
@@ -22,15 +27,66 @@ def test_PythonFormatString():
     assert placeables == ["{ name }"]
 
 
+def test_PythonFormattingVariable():
+    input = "My %(name)d is Luka."
+    placeables = get_placeables(input)
+
+    assert placeables == ["%(name)d"]
+
+
 def test_FluentTerm():
     input = "My { -name } is Luka."
     placeables = get_placeables(input)
 
     assert placeables == ["{ -name }"]
 
 
+def test_FluentParametrizedTerm():
+    input = "My { -name(foo-bar: 'now that's a value!') } is Luka."
+    placeables = get_placeables(input)
+
+    assert placeables == ["{ -name(foo-bar: 'now that's a value!') }"]
+
+
 def test_FluentFunction():
     input = "My { NAME() } is Luka."
     placeables = get_placeables(input)
 
     assert placeables == ["{ NAME() }"]
+
+
+def test_JavaFormattingVariable():
```

Comment:
These are succeeding because they're matching `PythonFormatString`, not `JavaFormattingVariable`.